README.md

```diff
@@ -1,15 +1,15 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
 # Project Name
-
-> Description the project.
+  Hello microverse
+> 
```

Comment:
- [ ] Please add a bit description of your project.

Reply:
### Added changes

- [ ] Added local storage